site/content/en/docs/manual/advanced/formats/format-lfw.md

```diff
@@ -0,0 +1,77 @@
+---
+linkTitle: 'LFW'
+weight: 17
+---
+
+# [LFW](http://vis-www.cs.umass.edu/lfw/)
+
+- Format specification available [here](http://vis-www.cs.umass.edu/lfw/README.txt)
+
+- Supported annotations: tags, points.
+
+- Supported attributes:
+
+  - `negative_pairs` (should be defined for labels as `text`):
+    list of image names with mismatched persons.
+  - `positive_pairs` (should be defined for labels as `text`):
+    list of image names with matched persons.
+
+# Import LFW annotation
+
+The uploaded annotations file should be a zip file with the following structure:
+
+```bash
+<archive_name>.zip/
+    └── annotations/
+        ├── landmarks.txt # list with landmark points for each image
+        ├── pairs.txt # list of matched and mismatched pairs of person
+        └── people.txt # list of persons name
```

Comment:
Isn't the file optional now?

Reply:
Yes, it's. Added clarification about it.